src/import/service.rs

```diff
@@ -1,12 +1,12 @@
 // Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
 
 use futures::Future;
-use grpcio::{RpcContext, RpcStatus, RpcStatusCode, UnarySink};
+use grpcio::{RpcContext, RpcStatus, RpcStatusCode::*, UnarySink};
```

Comment:
Could we avoid the glob import of enum variants? I think it is a bad practice because it makes code less readable (except for very well known enums like `Option`)

Reply:
It's a module rather than enum. It's generated by rust-bindgens.

Reply:
Ah, OK, shame it doesn't stick to the naming conventions I guess